pages/styles/university-based.css

```diff
@@ -0,0 +1,93 @@
+.title{
+    margin: 4vh;
+    color: #000;
+    text-align: center;
+    font-family: Poppins;
+    font-size: 30px;
+    font-weight: 600;
+}
+.title-about{
+    text-align: center;
+    font-family: Poppins;
+    font-size: 20px;
+    font-weight: 400;
+    margin: 2vh;
+}
+
+.universities-container{
+    display: flex;
+    flex-direction: column;
+    border-top:  2px solid #D9D4D7;
+    padding: 10px 5px 3px 3px;
+    margin-top: 5vh;
+    margin-left: 20%;
+    margin-right: 20%;
+    margin-bottom: 12svh;
+}
+.Organization-1{
+    display: flex;
+    flex-direction: row;
+}
+
+.Organization{
+    margin-top: 2vh;
+    padding: 10px 5px 3px 3px;
+    border-top:2px solid #D9D4D7;
+    display: flex;
+    flex-direction: row;
+}
+
+.org-logo {
+    margin-left: 3vh;
+    width: 100px;
+    height: 100px;
+    border-radius: 100px;
+    background: lightgray 50% / cover no-repeat;
+}
+
+.org-name{
+    position: relative;
+    margin-top: 6vh;
+    display: flex;
+    width: 680px;
+    height: 30px;
+    flex-direction: column;
+    justify-content: center;
+    margin-left: 3vh;
+    color: #21536C;
+    font-size: 20px;
+    font-weight: 700;
+    text-decoration: underline;
+    cursor: pointer;
+}
```

Comment:
Good thing you made the classes reusable thus, reducing the amount of CSS needed. I was expecting you to reuse a similar style found in the Search Results page.

Below is the code which you can find the same in the page's HTML.
```
<a class="search-result" href="">
    <span class="search-result__title">Page title asd asd</span>
    <span class="search-result__desc"
        >asdasdsadsadsadasdasdasda asd ass ad as asd asd as sa
        dsa dsa dsad asd asd
    </span>
    <img
        class="search-result__chevron"
        alt=""
        src="assets/icons/chevron_link.svg" />
</a>

Would it be better if we can have one style for the search results and organization links? 
```

Reply:
yes nong I think mas better nga one style lng